packs/equipment-effects/effect-apricot-of-bestial-might.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Apricot of Bestial Might",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p>You gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage.</p>"
```

Comment:
The trait should be `deadly-d12`. In addition, the resistance and clumsy could be automated.